getnacelle/contentful-app-nacelle
# refactor: preserve app state

**What This PR Does**
- fixes bug where updating config resets appearance of all content models using the nacelle app
- updates packages to expose required method `sdk.app.getCurrentState`

**How To Test**
- clone the repo
- `yarn start`
- open Prarie Wind Apparel experimental space
- click on `Apps > Manage Apps` and click on `Nacelle [Local Dev]`
- update the endpoint from `v2` to `v3` or vice versa
- open up `Content Models > Collection Grid` and click on Setting next to `Collection Handle`
- ensure that the field's appearance is still set to `Nacelle [Local Dev]`
